examples/basics/plotting/scatter_histogram.py

```diff
@@ -0,0 +1,24 @@
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vispy: gallery 1
+
+"""
+A scatter plot of 2D points with matching histograms.
+"""
+
+import numpy as np
+
+from vispy import plot as vp
+
+n = 100000
+data = np.random.randn(2, n)
+color = (0.8, 0.25, 0.)
+n_bins = 100
+
+fig = vp.Fig(show=False)
+fig[0:4, 0:4].plot(data, width=0, face_color=color + (0.05,), edge_color=None)
+fig[4, 0:4].histogram(data[0], bins=n_bins, color=color, orientation='h')
+fig[0:4, 4].histogram(data[1], bins=n_bins, color=color, orientation='v')
+
+if __name__ == '__main__':
+    fig.show(run=True)
```

Comment:
It is normal. The compound visuals do not yet support the `Visual.attach` API, which is necessary for clipping. That's in my current list of projects.